source/client-side-operations-timeout/etc/templates/deprecated-options.yml.template

```diff
@@ -0,0 +1,233 @@
+description: "operations error if timeoutMS is used with a deprecated timeout option"
+
+schemaVersion: "1.2"
+
+initialData:
+  - collectionName: &collectionName coll
+    databaseName: &databaseName test
+    documents: []
+
+tests:
+  # One-off tests to assert that drivers error if both maxCommitTimeMS and timeoutMS are set.
+
+  - description: "abortTransaction errors if maxCommitTimeMS and timeoutMS are both specified"
```

Comment:
Good catch. The transactions spec says:

> startTransaction SHOULD report an error if the driver can detect that transactions are not supported by the deployment.

To avoid adding too many special cases to the template, I set the min server version to 4.2 for the test file as that's the lowest version which supports transactions.